Source/Core/VideoBackends/Software/Tev.cpp

```diff
@@ -690,14 +544,15 @@ void Tev::Draw()
       DrawAlphaCompare(ac, inputs);
 
     if (ac.clamp)
-      Reg[u32(ac.dest.Value())][ALP_C] = Clamp255(Reg[u32(ac.dest.Value())][ALP_C]);
+      Reg[ac.dest][ALP_C] = Clamp255(Reg[ac.dest][ALP_C]);
     else
-      Reg[u32(ac.dest.Value())][ALP_C] = Clamp1024(Reg[u32(ac.dest.Value())][ALP_C]);
+      Reg[ac.dest][ALP_C] = Clamp1024(Reg[ac.dest][ALP_C]);
 
 #if ALLOW_TEV_DUMPS
     if (g_ActiveConfig.bDumpTevStages)
     {
-      u8 stage[4] = {(u8)Reg[0][RED_C], (u8)Reg[0][GRN_C], (u8)Reg[0][BLU_C], (u8)Reg[0][ALP_C]};
+      u8 stage[4] = {(u8)Reg[cc.dest][RED_C], (u8)Reg[cc.dest][GRN_C], (u8)Reg[cc.dest][BLU_C],
+                     (u8)Reg[ac.dest][ALP_C]};
```

Comment:
Why keep using the array syntax here/above when you started using .r/.g/.b/.a elsewhere?

Reply:
I'm not sure. This one might have been because this code is only enabled in debug mode, but there were two places where I explicitly used `[ALP_C]` when I could have used `.a` instead:

https://github.com/dolphin-emu/dolphin/blob/e983c18e736581c41744ce98d5ac4d575a459b99/Source/Core/VideoBackends/Software/Tev.cpp#L546-L549

https://github.com/dolphin-emu/dolphin/blob/e983c18e736581c41744ce98d5ac4d575a459b99/Source/Core/VideoBackends/Software/Tev.cpp#L160

(Both of these are odd; for the first one, I'd already changed the color code; for the second one, I'd changed `DrawAlphaCompare` but not `DrawAlphaRegular`. `DrawColorRegular`/`DrawColorCompare` do still need to use that indexing due to iterating over all colors).

I'll replace them.